packages/storybook/package.json

```diff
@@ -63,6 +63,7 @@
     "rimraf": "6.0.1",
     "shadow-dom-testing-library": "1.11.2",
     "storybook": "8.2.7",
+    "storybook-addon-pseudo-states": "4.0.2",
```

Comment:
Met deze Storybook addon kunnen de states geforceerd worden in een story.
https://storybook.js.org/addons/storybook-addon-pseudo-states

Reply:
Ga ik even verder naar kijken, focus lijkt nog niet helemaal goed te werken.

Reply:
![Screenshot 2024-10-21 at 13 26 53](https://github.com/user-attachments/assets/83a60b90-b9da-4170-a628-5f26cdfd6346)
Met `pseudo: { focus: true, focusVisible: true },` is de outline wel te zien. Of miste je naast de outline ook nog wat anders aan de focus state?

Reply:
👍